doc/rst/lib_board_support.rst

```diff
@@ -78,10 +109,93 @@ form of a task which is run on a thread placed on the I²C tile and is controlle
 from the other tile where I²S resides. The cross-tile channel must be declared at the top-level
 XC main function. The included examples provide a reference for this using both XC and C.
 
-.. note::
-    Sample XN files (hardware description for compiler) are provided in ``/xn_files`` in the cases
-    where the compiler doesn't support the platform directly. These can be copied into your
-    application directory where needed. See ``/examples`` for usage.
+|newpage|
+
+********************
+Example Applications
+********************
+
+Some simple example applications are provided in order to show how to use `lib_board_support`.
+
+Simple C Usage
+==============
+
+The applications `app_evk_316_simple_c` and `app_xk_audio_316_mc_simple_c` provide a bare-bones
+application where the hardware setup is called from C.
+
+These applications run on the `XK-EVK-XU316` and `XK-AUDIO-316-MC` boards respectively.
+
+They show how to use the cross-tile communications in conjunction with the I²C master server.
```

Comment:
slowly moving to "controller". Suggest "I2C controller (master)" for first usage then use "controller" for the rest of the doc